console/console-init/ui/src/components/DropdownWithToggle/DropdownWithToggle.tsx

```diff
@@ -34,16 +35,19 @@ export interface IDropdownWithToggleProps
   toggleIcon?: React.ReactElement<any>;
   component?: React.ReactNode;
   toggleClass?: string;
-  value?: string;
-  onSelectItem?: (value: string) => void;
+  value: string;
+  onSelectItem?: (value: string, event?: any) => void;
   dropdownItemIdPrefix?: string;
   dropdownItemClass?: string;
   isDisplayLabelAndValue?: boolean;
+  isLabelAndValueNotSame?: boolean;
```

Comment:
Just one grammatical suggestion I  would like to give: 

`isLabelAndValueNotSame` -> `areLabelAndValueNotSame` 
`isDisplayLabelAndValue` -> `shouldDisplayLabelAndValue`

What do you feel?

Reply:
Agreed on second point but not sure about first, i never seen boolean property start with prefix are. i always seen prefix "is". Lets keep it as it as.
Below points sounds good.
isDisplayLabelAndValue -> shouldDisplayLabelAndValue